.github/workflows/ci.yml

```diff
@@ -109,6 +116,35 @@ jobs:
           path: ./coverage/
           name: coverage
 
+  ef-tests:
+    runs-on: ubuntu-latest-32-cores
+    needs: build
+    steps:
+      - name: Checkout ef-tests
+        uses: actions/checkout@v3
+        with:
+          repository: kkrt-labs/ef-tests
+      - name: Rust cache
+        uses: Swatinem/rust-cache@v2
+        with:
+          cache-on-failure: "true"
+      - name: setup
+        run: make setup && make pull-kakarot
```

Comment:
we don't need to pull kakarot anymore, do we?

Reply:
removed `make pull-kakarot`, resolved.